packages/paste-website/src/pages/layout/grid/index.mdx

```diff
@@ -313,9 +305,50 @@ A Grid can be nested inside another Grid. For example, you can split a column in
 
 ### Responsive columns
 
-Grid column `span` and `offset` props can be changed based on the theme breakpoints. Both `span` and `offset` can accept responsive values, for example `span={[6, 8, 2] }`.
-This would set a columns `span` to be 6 columns wide on mobile, 8 columns wide on tablet, and 2 columns wide on desktop. The `offset` prop works in a similar fashion.
-For a visual example, [see the Storybook example](https://main--5e53448165911c0022e68c74.chromatic.com/?path=/story/layout-grid--grid-2-column-responsive) and resize the browser.
+Grid column `span`, `offset`, and `gutter` props accept accept responsive values, allowing them to be changed on theme breakpoints.
+For example, passing `span={[6, 8, 2] }` to a `Column`component sets the column `span` to be 6 columns wide on mobile, 8 columns wide on tablet, and 2 columns wide on desktop. The `offset`, `gutter`, and `vertical` props work in a similar fashion.
+Check out the examples below and resize the browser to see the responsive values in action.
+
+While you can achieve a one column responsive layout by changing each column `span` to 12, you lose out on the ability to have vertical spacing between each row. In this case, we recommend using the vertical prop as it will continue to apply the gutter spacing
```

Comment:
```suggestion
When creating responsive layouts with the Grid, it is important to remember to control the `vertical` prop accordingly. At the breakpoints you wish to stack the columns, set `vertical` to equal true. This will apply the column gutters vertically, rather than horizontally.
```

Reply:
@SiTaggart I replaced the text here, and moved a slightly reworded call out about setting span to 12 in a callout. LMK what you think, can easily remove.

Reply:
Here is is with the callout 👇 

<img width="816" alt="Screen Shot 2021-08-11 at 6 43 48 AM" src="https://user-images.githubusercontent.com/18120906/129040326-9f280b1a-07f8-4d4b-8ca9-3e8be68401cf.png">

Reply:
(btw I see that there is a mis-formatted word "span"), if you want me to include this I can fix it when I restore the commit)

Reply:
Okey cool, This is ready to go then for another review. The most current commit doesn't have the callout.